src/cirrus/lib/statedb.py

```diff
@@ -522,3 +537,4 @@ def since_to_timedelta(cls, since: str) -> timedelta:
         hours = int(since[0:-1]) if unit == 'h' else 0
         minutes = int(since[0:-1]) if unit == 'm' else 0
         return timedelta(days=days, hours=hours, minutes=minutes)
+        
```

Comment:
You should probably configure your editor to add newlines to the end of files, as that is generally best practice.